src/packages/@teams/features/team-explorer/components/workspace-grid-view/WorkspaceGridView.svelte

```diff
@@ -0,0 +1,314 @@
+<script lang="ts">
+  import {
+    DoubleLeftIcon,
+    DoubleRightIcon,
+    LeftIcon,
+    RightIcon,
+  } from "$lib/assets/app.asset";
+  import type { Team } from "$lib/utils/interfaces";
+  import type { WorkspaceDocument } from "@app/database/database";
+  import Button from "@library/ui/button/Button.svelte";
+  import WorkspaceGrid from "$lib/components/dashboard/workspace-grid/WorkspaceGrid.svelte";
+
```

Comment:
not required to import this component  from $lib that represents deprecate. please push this component to newer folder structure

Reply:
Done, although didn't removed the old files as their were using in .old files so will remove them later